### ISSUE TITLE
Cannot find module 'inquirer'

### ISSUE THREAD

**Issue by abhishiv** [victorgarciaesgi/simple-graphql-to-typescript#61]
```
Error: Cannot find module 'inquirer'
Require stack:
- /Users/abhishiv/work/gratico/gratico/node_modules/simple-graphql-to-typescript/dist/rc/index.js
- /Users/abhishiv/work/gratico/gratico/node_modules/simple-graphql-to-typescript/dist/sgts-bin.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/abhishiv/work/gratico/gratico/node_modules/simple-graphql-to-typescript/dist/rc/index.js:10:36)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/abhishiv/work/gratico/gratico/node_modules/simple-graphql-to-typescript/dist/rc/index.js',
    '/Users/abhishiv/work/gratico/gratico/node_modules/simple-graphql-to-typescript/dist/sgts-bin.js'
  ]
```

**Comment by victorgarciaesgi** [comment]
Hi @abhishiv thanks for the bug report, it was a dependency bug, `inquirer` should be in "dependencies".
Should be fixed in `0.10.11` !

**Comment by victorgarciaesgi** [comment]
@abhishiv Is it fixed?
